react-frontend/src/containers/App/index.js

```diff
@@ -8,12 +8,14 @@ import BreadCrumbs from '../../components/Nav/breadcrumbs';
 
 import '../../css/global.css';
 import { RouteBody } from '../../components/StyleComponents/pageContent';
+import CovidBanner from '../../components/Nav/covidbanner';
 
 function App() {
   return (
     <div className="App">
       <Router>
         <NavBar />
+        <CovidBanner />
```

Comment:
Just wondering why CovidBanner is placed after, and not before, NavBar here?

Reply:
I don't think placing matters here.